### PR TITLE
cargo: fill required fields for publishing

### PR DESCRIPTION
**Pull request by Wenzel**
Fill essential keys required for a publication on crates.io

help fixing #108

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,12 @@ name = "microvmi"
 version = "0.1.0"
 authors = ["Mathieu Tarral <mathieu.tarral@protonmail.com>"]
 edition = "2018"
+license = "GPL-3.0-only"
+description = "A cross-platform, unified, low-level VMI API on top of hypervisors"
```
Comment (Wenzel) [inline Cargo.toml:7]: @rageagainsthepc I'd like your opinion on the description.
how do you feel when reading it ?
can we do better ?
my english is not incredible.

Comment (rageagainsthepc) [reply Cargo.toml:6]:
```suggestion
description = "A cross-platform, unified, low-level VMI API supporting multiple hypervisors"
```
Perhaps?

Comment (rageagainsthepc) [reply Cargo.toml]: Hm, maybe you can scratch that idea. The word "unified" already gives away that multiple hypervisor are supported. lgtm

Comment (Wenzel) [reply Cargo.toml]: Unified means to me "abstraction layer", so I'll keep it for now

**Pull request merged**